Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -166,6 +168,11 @@ public void SetLogLevel(int logLevel)
         AppCenter.LogLevel = Microsoft.AppCenter.Unity.LogLevel.Verbose + logLevel;
     }
 
+    public void setStartupMode()
+    {
+        AppCenter.StartupType = (StartupType)Enum.Parse(typeof(StartupType), Microsoft.AppCenter.Unity.StartupType);
```

Comment:
Todo: find a way to save this in native preferences. Use `#if UNITY_ANDROID && !UNITY_EDITOR ... #elseif UNITY_IOS && !UNITY_EDITOR ...`